docs/Drawer.md

```diff
@@ -278,11 +276,34 @@ The `DrawerLayout` component is used to provide the appropriate resizing behavio
     <img width="100%" style="max-width: 600px" alt="DrawerLayout with labels" src="./images/drawerLayout.png">
 </div>
 
+### DrawerLayout API
+
+<div style="overflow: auto;">
+
+| Prop Name          | Description                                           | Type              | Required  | Default |
+|--------------------|-------------------------------------------------------|-------------------|-----------|---------|
+| children           | Page's body content                                   | `React.ReactNode` | yes       |         |   
+| drawer             | Drawer component to be embedded                       | `JSX.Element`     | yes       |         |   
+| variantBreakpoints | Change the drawer variant according to the breakpoint | `{[key: string]: DrawerProps['variant']}`| no | See below |   
```

Comment:
perfect opportunity for an example column :P

Reply:
Will be a huge column, sir. I prefer not.

Reply:
only kidding

Reply:
r/wooooooosh